tests/unit/suites/tracer/linux/mfx_tracer_test_cases_libs.cpp

```diff
@@ -41,7 +41,7 @@ auto func_case_list = ::testing::Values(
 TEST_P(TracerLibsTestParametrizedFunc, ShouldSucceedForSeeminglyGoodMockLibrary)
 {
     MockCallObj& mock = *g_call_obj_ptr;
-    mfxStatus sts;
+    mfxStatus sts = MFX_ERR_NONE;
```

Comment:
so the issue would be if ```func``` is not ```MFX_INIT``` and not ```MFX_INIT_EX```. I think in this case we should still fail with error, so for this particular case we should use different initialization status, not ```MFX_ERR_NONE```

Reply:
In this test `func` have only `MFX_INIT` and `MFX_INIT_EX` values
https://github.com/Intel-Media-SDK/MediaSDK/blob/40ab977d9b50865f2248fd3b628e07738c9633a4/tests/unit/suites/tracer/linux/mfx_tracer_test_cases_libs.cpp#L37-L39

Reply:
then you don't need to fix it. Personally I believe that better to fix it, because calling conditions may change at some point.

Static analysis complains on this code because it sees possible control path where ```sts``` is not set. If you decide to fix this "issue" I suggest to assume that any value can come to this function at ```func```

Reply:
Thanks